manifests/kustomize/base/pipeline/ml-pipeline-ui-role.yaml

```diff
@@ -14,6 +14,13 @@ rules:
   - create
   - get
   - list
+- apiGroups:
```

Comment:
you should reflect the same changes here too: https://github.com/kubeflow/pipelines/blob/master/manifests/gcp_marketplace/chart/kubeflow-pipelines/templates/pipeline.yaml#L148

Reply:
rebased and updated the helm chart.